InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java

```diff
@@ -126,19 +127,35 @@ public DriveSubsystem(Config config, NavigationSubsystem navigationSubsystem, Vi
         trajectoryConfig.addConstraint(kinematicsConstraint);
         trajectoryConfig.addConstraint(voltageConstraint);
 
-        autoTrajectory = TrajectoryGenerator.generateTrajectory(
-             // Start at the origin facing the +X direction
+        pickupTrajectory = TrajectoryGenerator.generateTrajectory(
+            // Start at the origin facing the +X direction
             new Pose2d(0, 0, new Rotation2d(0)),
-            // Pass through these two interior waypoints, making an 's' curve path
+            // Pass through these two interior waypoints
             List.of(
-                new Translation2d(2, 0.5),
-                new Translation2d(4, 0.75)
+                new Translation2d(1.5, 1.2),
+                new Translation2d(2, 1.5),
+                new Translation2d(4, 1.6)
             ),
-            // End 3 meters straight ahead of where we started, facing forward
-            new Pose2d(5, 1, new Rotation2d(0)),
+            // End 5 meters ahead and 1 meter over of where we started, facing forward
+            new Pose2d(5, 1.75, new Rotation2d(0)),
             trajectoryConfig
         );
 
+        TrajectoryConfig trajectoryConfigReversed = new TrajectoryConfig(
```

Comment:
We don't need a separate TrajectoryConfig object for reversed trajectoryConfig. We can set the reversed flag and pass in the config to the next generateTrajectory function. It works.